### ISSUE TITLE
can't read some blocks, eg block 1564403 https://testnet3.aleoscan.io/block?h=1564403

### ISSUE THREAD

**Issue by changqa** [HarukaMa/aleo-explorer#21]
https://testnet3.aleoscan.io/block?h=1564403
returns 500 error

**Comment by HarukaMa** [comment]
Fix in the progress but it's currently blocked by another bug that affected database integrity. Might take a while to fix.

**Comment by WSbaikaishui** [comment]
on testnet-beta, can't read 539
adding block Block 539 (ab17q9sytt6lqt4j...)
expected: [<aleo_types.vm_block.UpdateKeyValue object at 0x7f0d52f91450>, <aleo_types.vm_block.UpdateKeyValue object at 0x7f0d52f90c90>, <aleo_types.vm_block.UpdateKeyValue object at 0x7f0d52f90f10>, <aleo_types.vm_block.UpdateKeyValue object at 0x7f0d52f91e10>, <aleo_types.vm_block.UpdateKeyValue object at 0x7f0d52f91c50>, <aleo_types.vm_block.UpdateKeyValue object at 0x7f0d52f91e90>]
actual: [{'type': <Type.UpdateKeyValue: 2>, 'program_name': 'credits.aleo', 'mapping_id': Field(2855157744830843716005407030207142101853521493742120919939436395872133863104), 'key_id': Field(2927729380015131696544723331679864896636763111582109488439917490193194514968), 'value_id': Field(260980685651009611918596170818859921204353249451250740631528542657215795011), 'mapping_name': account, 'key': aleo1u4l0ha0w4pjcxj5fnl7xzyl94wp5z5geffwq6hg5ukg4jvf8eq9qxw0dcn, 'value': 999999499775u64, 'height': 539, 'from_transaction': True}]
execute error: value -9000000193996 out of range for u64, at transition #0, instruction "sub r18 r3 into r19"

**Comment by HarukaMa** [comment]
Please open an new issue if there are other sync issues
